docs/index.rst

```diff
@@ -73,6 +73,13 @@ or watch the `CompuCell3D Workshop Python tutorial videos <https://www.youtube.c
     maboss
 
 
+.. toctree::
+    :caption: Complete Tutorials
+    :hidden:
+
+    example_compartmental_elongated_cells_tutorial
```

Comment:
Does the "Real-World Examples" display for you when you open documentation? It does not for me.

https://compucell3dreferencemanual.readthedocs.io/en/latest/

We might need to do some upgrades to the index but I agree we should group it

Reply:
It's hidden for me too. Could you please adjust this part of index.rst
```
.. toctree::
   :maxdepth: 1
   :hidden:
   :caption: Real-world Examples

    section_real_world_examples
```

so that either `maxdepth` is 2 or `hidden` is removed?

Reply:
Done - it worked, thanks!